Mathlib/Data/Fin/Basic.lean

```diff
@@ -537,6 +538,18 @@ protected theorem zero_add [NeZero n] (k : Fin n) : 0 + k = k := by
 instance [NeZero n] : OfNat (Fin n) a where
   ofNat := Fin.ofNat' a (NeZero.pos n)
 
+instance inhabited (n : ℕ) [NeZero n] : Inhabited (Fin n) :=
```

Comment:
Do we automatically get `Inhabited (Fin (n+1))` from this?

Reply:
No, we have no `NeZero (n + 1)` instance. Note that this instance isn't new, I just moved it from another file.

Reply:
Then it seems to me that this PR is removing something, because `Inhabited (Fin (n+1))` is automatically inferred in current mathlib?

Reply:
Sorry, I misread and thought you said `1+n`. Yes, `n+1` is inferred.

Reply:
The answer to your original question is yes.